text/0115-mixed-shadow-mode.md

```diff
@@ -50,28 +51,30 @@ export default class extends LightningElement {
 ```
 
 This mode will result in the whole component subtree to operate in either native shadow mode or
-synthetic shadow mode. If the browser supports Shadow DOM the shadow mode will be native. If the
-browser does not support Shadow DOM (e.g., IE11) the shadow mode will be synthetic. The test to
+synthetic shadow mode. If the browser supports Shadow DOM, the shadow mode will be native. If the
+browser does not support Shadow DOM (e.g., IE11), the shadow mode will be synthetic. The test to
 determine Shadow DOM support will be to check whether `window.ShadowRoot` is defined.
 
 ### Opting out
 
-A component that is extending a component class that opted in to native shadow mode can opt out of
-native shadow mode by setting its own static `shadowSupportMode` property to `default`; however,
+A component that is extending a component class that opts in to native shadow mode can opt out of
+native shadow mode by setting its own static `shadowSupportMode` property to `reset`; however,

```

Comment:
I proposed yesterday the name `unset` in reference to the [CSS value](https://developer.mozilla.org/en-US/docs/Web/CSS/unset). I am also ok with `reset` as both read better than `default`.